fenix/app/onboarding.fml.yaml

```diff
@@ -0,0 +1,119 @@
+---
+features:
+
+  juno-onboarding:
+    description: A feature that shows juno onboarding flow.
+
+    variables:
+      enabled:
+        description: if true, juno onboarding is shown to the user.
+        type: Boolean
+        default: false
+      cards:
+        description: Collection of user facing onboarding 'views'.
+        type: Map<String, OnboardingCardData>
+        default:
+            default-browser:
+              card-type: default-browser
+              telemetry:
+                id: "default"
+                sequence-position: "1"
```

Comment:
💭 This works, but I'm a afraid if this is changed by mistake by the experiment creators. For instance, if the ordering is updated without updating the sequence-position, it's a "bug" that results in wrong telemetry.

An alternative would be to get the `sequence-position` from position of the card in the list, advantage would be that it can be tested and that functionality won't need to change regardless of the number of cards added/removed. 

It does make sense to keep the `id` here as that's based on the card itself and whenever a new card is added, it makes people think of figuring out the telemetry id, although there might be risk of someone adding a really long id, IIRC, there's a glean extras character limit 🤔

Reply:
> IIRC, there's a glean extras character limit

Interesting, I did not know this!